src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -477,16 +637,30 @@ void goto_analyzer_parse_optionst::help()
     " goto-analyzer [-h] [--help]  show help\n"
     " goto-analyzer file.c ...     source file names\n"
     "\n"
-    "Analyses:\n"
+    "Task options:\n"
+    " --show                       display the abstract domains\n"
+    " --verify                     use the abstract domains to check assertions\n" // NOLINT(*)
+    " --simplify file_name         use the abstract domains to simplify the program\n" // NOLINT(*)
     "\n"
-    " --taint file_name            perform taint analysis using rules in given file\n"
-    " --unreachable-instructions   list dead code\n"
-    " --intervals                  interval analysis\n"
-    " --non-null                   non-null analysis\n"
+    "Abstract interpreter options:\n"
+    " --flow-sensitive             use flow-sensitive abstract interpreter\n"
+    " --concurrent                 use concurrent abstract interpreter\n"
```

Comment:
I think this is misleading: it's a concurrency-aware AI, not a concurrently-executing one.